src/hub/Hub.sol

```diff
@@ -474,6 +517,7 @@ contract Hub is Auth, Multicall, IHub, IHubGatewayHandler {
     ) external auth {
         accounting.unlock(poolId);
         address poolCurrency = hubRegistry.currency(poolId).addr();
+        // TODO(wischli): Investigate whether priceAssetPerShare is the correct provider from CV side
```

Comment:
@mustermeiszer Can you quickly validate that we indeed want to use the `priceAssetPerShare` here? Wondering because we set the price for base `assetId` and quote `poolCurrency` in the line below instead of the provided base `shareToken` and quote `assetId`. Don't we actually want `pricePoolPerAsset` here?

Reply:
To provide more context: In `SyncRequests._updateHoldings` we provide `priceAssetPerShare` as the `pricePerUnit` for `balanceSheet.deposit` which then sends `UpdateHoldingAmount` with that price: https://github.com/centrifuge/protocol-v3/pull/184/files#diff-189528fffa0deeddc2713fe3302b7cd454d4e87012e28a6bd88562adabd2d63eR268

Reply:
We need to use `pricePoolPerAsset`. See comments in `SyncRequest.sol`

Reply:
Fixed